setup.py

```diff
@@ -45,7 +45,7 @@
     "pytest-runner>=2.7",
 ]
 
-install_requires = ["click>=7,<8", "requests==2.22.0"]
+install_requires = ["click>=7,<8", "requests==2.22.0", "pycurl>=7"]
```

Comment:
Travis CI fails with:

```
src/pycurl.h:202:30: fatal error: gnutls/gnutls.h: No such file or directory
```

You may need to amend `.travis.yml` to preinstall `gnutls` libraries.